php/libraries/NDB_Form.class.inc

```diff
@@ -206,10 +206,9 @@ class NDB_Form extends NDB_Page
     }
 
     /**
-     * Converts the results of this form to a JSON format to be retrieved
-     * with ?format=json
+     * {@inheritDoc}
```

Comment:
what does this do ?

Reply:
`{@inheritDoc}` is a convention in PHP recognized by phpcs/phpdoc/etc so that you don't need to copy the doc from the parent class.